src/app/containers/ArticleTimestamp/shouldDisplayLastUpdatedTimestamp.test.jsx

```diff
@@ -0,0 +1,137 @@
+import shouldDisplayLastUpdatedTimestamp, {
+  hasBeenUpdated,
+  publishedAndUpdatedToday,
+} from './shouldDisplayLastUpdatedTimestamp';
+import { timestampGenerator, sameDayTimestampsGenerator } from './testHelpers';
+
+describe('shouldDisplayLastUpdatedTimestamp functions', () => {
+  let originalDate;
+
+  beforeEach(() => {
+    originalDate = Date.now;
+  });
+
+  afterEach(() => {
+    Date.now = originalDate;
+  });
+
```

Comment:
You could add a nested describe here for the hasBeenUpdated function, that way, the following 2 tests would read `it('should...')`